src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -74,6 +74,15 @@
         libldb                  \
         libtalloc               \
         libtevent
+  - name: Install packages required for passkey testing
+    dnf:
+      state: present
+      name:
+        - gcc
+        - openssl-devel
+        - umockdev
+    when: (ansible_facts['distribution'] == "CentOS" and ansible_distribution_major_version | int >= 9) or
+          (ansible_facts['distribution'] == "Fedora")
```

Comment:
It would be better to add new fact to https://github.com/SSSD/sssd-ci-containers/tree/master/src/ansible/roles/facts/tasks

E.g.

```yaml
passkey_support: Yes|No
```

End then use just `when: passkey_support`

Reply:
I hope I did it correctly. Will need to check the results later.

By the way, thanks for the lesson about facts ;)